src/py42/modules/archive.py

```diff
@@ -7,6 +7,114 @@ def __init__(self, archive_accessor_manager, archive_client):
         self._archive_accessor_manager = archive_accessor_manager
         self._archive_client = archive_client
 
+    def get_by_archive_guid(self, archive_guid):
+        """Gets single archive information by GUID.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            archive_guid (int): The GUID for the archive.
+
+        Returns:
+            :class:`py42.response.Py42Response`: A response containing archive
+            information.
+        """
+        uri = u"/api/Archive/{}".format(archive_guid)
+        return self._archive_client.get_single_archive(uri)
+
+    def get_by_device_guid(self, device_guid):
+        """Gets archive information for a device.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            device_guid (int): The GUID for the device.
+
+        Returns:
+            generator: An object that iterates over :class:`py42.response.Py42Response`
+            objects that each contain a page of archives.
+        """
+        return self._archive_client.get(device_guid, u"backupSourceGuid")
+
+    def get_by_user_uid(self, user_uid):
+        """Gets archive information for a user.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            user_uid (int): The userUid for a user.
+
+        Returns:
+            generator: An object that iterates over :class:`py42.response.Py42Response`
+            objects that each contain a page of archives.
+        """
+        return self._archive_client.get_archives_from_value(user_uid, u"userUid")
+
+    def get_by_store_point_id(self, store_point_id):
+        """Gets archive information for a store point.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#Archive-get>`__
+
+        Args:
+            store_point_id (int): The storePointId for a storepoint.
+
+        Returns:
+            generator: An object that iterates over :class:`py42.response.Py42Response`
+            objects that each contain a page of archives.
+        """
+        return self._archive_client.get_archives_from_value(store_point_id,
```

Comment:
Some of the line wrapping on this file differs from how our formatters do it.

Reply:
draft PRs dont go through ci/cd. @DiscoRiver, be sure to read through `CONTRIBUTING.md` to get an environment set up that will perform all of your style checks, auto formatting, etc.

Reply:
Thanks! I forgot to run it! Everything passed now on my latest commits.